### ISSUE TITLE
Scroll if not visible

### ISSUE THREAD

**Issue by kristian-puccio** [fisshy/react-scroll#42]
Hi I'm thinking it would be useful to have a flag to only scroll if the element you are scrolling to isn't visible.

Are you into the idea? If so I could make a pull request.

**Comment by fisshy** [comment]
Yes sounds like a good feature - make sure you include som testing for it.
